iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/AbstractDisconnection.java

```diff
@@ -42,7 +45,7 @@ public void applyModification(Network network, boolean isPlanned, ReportNode rep
         // Disconnect the connectable
         boolean hasBeenDisconnected;
         try {
-            hasBeenDisconnected = connectable.disconnect(openableSwitches);
+            hasBeenDisconnected = connectable.disconnect(openableSwitches, side);
```

Comment:
The message added to the `ReportNode` should also include the side when it is present.

Reply:
done